### DISCUSSION TITLE
React Error Overlay

### DISCUSSION THREAD

**Discussion by chrislacey89** [parcel-bundler/parcel#6331] [Q&A] [answered]
I'm moving a project over from Create React App and I'm hoping to use Parcel. I can't seem to figure out how to get parcel to display a client side error overlay, similar to what you might have with CRA or Next. Instead I get a white screen and an error message in the console. Below are some pics that show what I'm talking about. I tried just about everything I can think of including adding the `--log-level`  flag, but I've had no luck.... Am I missing something obvious? Top pic is an example of the error overlay from a Next app. 

![Screen Shot 2021-05-22 at 3 40 58 PM](https://user-images.githubusercontent.com/24979651/119239046-4a95bf80-bb14-11eb-9525-7f9131eb4e35.jpg)

![Screen Shot 2021-05-22 at 3 28 35 PM](https://user-images.githubusercontent.com/24979651/119238855-38675180-bb13-11eb-9da2-7c3b08e5dbf7.jpg)
![Screen Shot 2021-05-22 at 3 29 09 PM](https://user-images.githubusercontent.com/24979651/119238856-38ffe800-bb13-11eb-9b53-9546f6de3bf6.jpg)
![Screen Shot 2021-05-22 at 3 29 36 PM](https://user-images.githubusercontent.com/24979651/119238858-3ac9ab80-bb13-11eb-9093-6b4211898aa0.jpg)

**Comment by davidjb** [comment, marked as answer]
React Error Overlay was implemented as part of Parcel v2.6.0 (https://parceljs.org/blog/v2-6-0/#react-error-overlay)
